lib/searchkick/query.rb

```diff
@@ -490,10 +492,26 @@ def set_fields
           []
         elsif default_match == :exact
           raise ArgumentError, "Must specify fields to search"
+        elsif has_field_misspellings(options[:misspellings])
+          raise ArgumentError, "If you provide per-field misspelling rules you must also provide the fields to search."
```

Comment:
Let's do the `ArgumentError` check in `prepare` so we don't pollute `set_fields`.

Reply:
updated in 9e7a7e1101a6b1f241ee9332177e5889a44c0f24